code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -132,7 +132,13 @@
 	var/shouldwakeup = FALSE
 
 	///Domestication.
-	var/tame = 0
+	var/tame = FALSE
+	///What the mob eats, typically used for taming or animal husbandry.
+	var/food_type
+	///Starting success chance for taming.
+	var/tame_chance
+	///Added success chance after every failed tame attempt.
+	var/bonus_tame_chance
```

Comment:
🤔 I dislike guaranteed taming and especially the fact that this is global and not per-user is very iffy.

Reply:
what do you mean guaranteed taming

Reply:
the odds increase with every attempt, so its guaranteed to tame after like 3 tries

Reply:
it's up to whoever adds more tameable mobs to adjust the chances accordingly to not be as easy (and set the bonus to 0 if they so desire), as well as making the food item suitably rare - as for cows their only value is a gimmick so they are rather easy to tame

Reply:
I didn't actually do the math what I'm implying is that you Bob doe can feed a cow like 4 times, get unlucky but still move the chance to 85, then some random feeds the cow once and tames it.

Besides that, the fact that feeding something 6 times guarantees you tame it is eh.

Reply:
well it's either pure RNG, static progress or a bit of both that can be tuned either way, which is what I opted for

As I said if someone wants to give a mob a 1% base chance with a 0% increase every failure and require feeding them bluespace crystals they can do that also

This can always change later but I'm keen on keeping it as is for now, I used cows as an example just so I didn't have to deal with any balance concerns